lib/widgets/subscription_list.dart

```diff
@@ -0,0 +1,239 @@
+        child: Text("No streams found",
+          textAlign: TextAlign.center,
+          style: TextStyle(
+          color: const HSLColor.fromAHSL(1.0, 240, 0.1, 0.5).toColor(),
+          fontFamily: 'Source Sans 3',
+          fontSize: 18,
+          height: (20 / 18),
+        ).merge(weightVariableTextStyle(context)))));
+  }
+}
+
+class _SubscriptionListHeader extends StatelessWidget {
+  const _SubscriptionListHeader({
+    required this.context,
+    required this.label,
+  });
+
+  final BuildContext context;
+  final String label;
+
+  @override
+  Widget build(BuildContext context) {
+    return SliverToBoxAdapter(
+      child: ColoredBox(
+        color: Colors.white,
+        child: SizedBox(
+          height: 30,
+          child: Row(
+            children: [
+              const SizedBox(width: 16),
+              Expanded(child: Divider(
+                color: const HSLColor.fromAHSL(0.2, 240, 0.1, 0.5).toColor())),
+              const SizedBox(width: 8),
+              Text(label,
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  color: const HSLColor.fromAHSL(1.0, 240, 0.1, 0.5).toColor(),
+                  fontFamily: 'Source Sans 3',
+                  fontSize: 14,
+                  letterSpacing: 0.56,
+                  height: (16 / 14),
+                ).merge(weightVariableTextStyle(context))),
+              const SizedBox(width: 8),
+              Expanded(child: Divider(
+                color: const HSLColor.fromAHSL(0.2, 240, 0.1, 0.5).toColor())),
+              const SizedBox(width: 16),
+            ]))));
+  }
+}
+
+class _SubscriptionList extends StatelessWidget {
+  const _SubscriptionList({
+    required this.unreadsModel,
+    required this.subscriptions,
+  });
+
+  final Unreads? unreadsModel;
+  final List<Subscription> subscriptions;
+
+  @override
+  Widget build(BuildContext context) {
+    return SliverList.builder(
+      itemCount: subscriptions.length,
+      itemBuilder: (BuildContext context, int index) {
+        final subscription = subscriptions[index];
+        final unreadCount = unreadsModel!.countInStreamNarrow(subscription.streamId);
+        return SubscriptionItem(subscription: subscription, unreadCount: unreadCount);
+    });
+  }
+}
+
+@visibleForTesting
+class SubscriptionItem extends StatelessWidget {
+  const SubscriptionItem({
+    super.key,
+    required this.subscription,
+    required this.unreadCount,
+  });
+
+  final Subscription subscription;
+  final int unreadCount;
+
+  @override
+  Widget build(BuildContext context) {
+    final swatch = subscription.colorSwatch();
+    final hasUnreads = (unreadCount > 0);
+    return Material(
+      color: Colors.white,
+      child: InkWell(
+        onTap: () {
+          Navigator.push(context,
+            MessageListPage.buildRoute(context: context, narrow: StreamNarrow(subscription.streamId)));
+        },
+        child: SizedBox(height: 40,
```

Comment:
Probably a min-height is better, to accommodate large font sizes.

Reply:
As we talked in our pairing yesterday, I've switched this to not have a defined height, but added padding to the elements in this so that it results in 40px (and the `_SubscriptionListHeader` similarly for 30px) so that they can expand if the font size is larger.